lib/yggdrasil/player.ex

```diff
@@ -12,9 +13,8 @@ defmodule Yggdrasil.Player do
     Supervisor.add_player user_id, [owner_pid, push_msg]
   end
 
-  def run_cmd(user_id, cmd = %Message{type: :command}) do
-    player_pid = Registry.get_player user_id
-    GenServer.cast(player_pid, {:run_cmd, cmd})
+  def run_cmd(user_id, cmd) do
+    Command.execute cmd, user_id
```

Comment:
I was thinking the command would be executed by the room process. It would sort of be the coordinator between various things anyway and it would naturally serialize things like trying to pick up an item before someone else, or attacking something before it leaves. The room could also wrap the execute in a `try...catch` and forward the error back to the player. This is just my initial design idea.